test/.rubocop.yml

```diff
@@ -11,3 +11,6 @@ Style/ClassAndModuleChildren:
 # Some test models need methods defined on them in a #create_model block.
 Lint/NestedMethodDefinition:
   Enabled: false
+
+Layout/LineLength:
+  Enabled: false
```

Comment:
I'd prefer to keep this rule enabled and disable it only for user-facing messages with as few other exception as possible. Longer lines are harder to read.